apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -1,24 +1,57 @@
 /**
  * Generated by `createschema ticket.Ticket organization:Text; statusReopenedCounter:Integer; statusReason?:Text; status:Relationship:TicketStatus:PROTECT; number?:Integer; client?:Relationship:User:SET_NULL; clientName:Text; clientEmail:Text; clientPhone:Text; operator:Relationship:User:SET_NULL; assignee?:Relationship:User:SET_NULL; classifier:Relationship:TicketClassifier:PROTECT; details:Text; meta?:Json;`
  */
-const { createTestContact } = require('@condo/domains/contact/utils/testSchema')
-const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
-const { NUMBER_RE, UUID_RE, DATETIME_RE, makeClient, makeLoggedInAdminClient } = require('@core/keystone/test.utils')
-const { Ticket, createTestTicket, updateTestTicket } = require('@condo/domains/ticket/utils/testSchema')
-const { expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects, expectToThrowUserInputError } = require('@condo/domains/common/utils/testSchema')
-const { expectToThrowAccessDeniedErrorToObj } = require('@condo/domains/common/utils/testSchema')
-const { createTestOrganizationLink, createTestOrganizationWithAccessToAnotherOrganization } = require('@condo/domains/organization/utils/testSchema')
 const faker = require('faker')
-const { createTestOrganization } = require('@condo/domains/organization/utils/testSchema')
-const { createTestProperty, updateTestProperty } = require('@condo/domains/property/utils/testSchema')
-const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
-const { makeClientWithResidentAccessAndProperty } = require('@condo/domains/property/utils/testSchema')
-const { createTestOrganizationEmployeeRole } = require('@condo/domains/organization/utils/testSchema')
-const { updateTestOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
-const { sleep } = require('@condo/domains/common/utils/sleep')
 const dayjs = require('dayjs')
+const isEmpty = require('lodash/isEmpty')
+
+const conf = require('@core/config')
+const { NUMBER_RE, UUID_RE, DATETIME_RE, makeClient, makeLoggedInAdminClient } = require('@core/keystone/test.utils')
+const { find } = require('@core/keystone/schema')
```

Comment:
You should avoid using low-level database utils inside the tests. Use TestUtils for that.